Makefile

```diff
@@ -681,35 +683,35 @@ docker-build-e2e: ## Run docker-build-* targets for all the images with settings
 
 .PHONY: docker-build-core
 docker-build-core: ## Build the docker image for core controller manager
-	docker build --build-arg builder_image=$(GO_CONTAINER_IMAGE) --build-arg goproxy=$(GOPROXY) --build-arg ARCH=$(ARCH) --build-arg ldflags="$(LDFLAGS)" . -t $(CONTROLLER_IMG):${DOCKER_IMAGE_TAG}
+	docker build --build-arg builder_image=$(GO_CONTAINER_IMAGE) --build-arg golang_image=$(GOLANG_IMAGE) --build-arg goproxy=$(GOPROXY) --build-arg ARCH=$(ARCH) --build-arg ldflags="$(LDFLAGS)" . -t $(CONTROLLER_IMG):${DOCKER_IMAGE_TAG}
```

Comment:
Why additional build arg  golang_image need to be passed here? You already have a GO_CONTAINER_IMAGE ?